docs/configuration.md

```diff
@@ -366,12 +379,256 @@ If you're interested in testing hover, focus, and active states with Happo, you
 may also want to use the
 [`applyPseudoClasses` option](#target-applypseudoclasses).
 
+### Target `outgoingRequestHeaders`
+
+Add additional headers to outgoing requests from the browser. This is useful if
+you need to tell a CDN or other service that the request originates from a Happo
+run, or if you need to pass authentication headers.
+
+**Note:** This option only applies to desktop browsers (Chrome, Firefox, Edge,
+Safari).
+
+```js title="happo.config.ts"
+import { defineConfig } from 'happo';
+
+export default defineConfig({
+  targets: {
+    chrome: {
+      browserType: 'chrome',
+      viewport: '1024x768',
+      outgoingRequestHeaders: [
+        { name: 'X-Happo-Run', value: 'true' },
+        { name: 'Authorization', value: 'Bearer token123' },
+      ],
+    },
+  },
+});
+```
+
 ## `project`
 
 If you have multiple projects configured for your happo.io account, specify the
 name of the project you want to associate with. If left empty, the default
 project will be used.
 
+```js title="happo.config.ts"
+import { defineConfig } from 'happo';
+
+export default defineConfig({
+  project: 'my-project-name',
+  // ... rest of config
+});
+```
+
+## `integration`
+
+Specify the type of integration you're using with Happo. The integration type
+determines how Happo discovers and renders your components.
+
+### `integration.type`
+
+The type of integration. Supported values:
+
+- `'storybook'` - For Storybook integrations
+- `'cypress'` - For Cypress E2E test integrations
+- `'playwright'` - For Playwright E2E test integrations
+- `'custom'` - For custom bundle integrations
+- `'pages'` - For full-page screenshot integrations
+
+### `integration.configDir` (Storybook only)
```

Comment:
Instead of adding the integration only type in parentheses, what do you think about adding a heading like `### Storybook Integration Options` and then nesting the options below that?